signer-cloud-server/src/test/java/com/wultra/signercloud/server/signer/SignerControllerIntTest.java

```diff
@@ -97,7 +99,7 @@ void testCreateUpdateWhenOperationFailsThenFailResponseIsReturned() throws Excep
         final var request = new CreateUpdateSignerRequest(DUMMY_EXTERNAL_SIGNER_ID, DUMMY_USER_ID, DUMMY_CSR);
 
         // when
-        var mvcResult = mockMvc.perform(post(CREATE_UPDATE_SIGNER_ENDPOINT)
+        final var mvcResult = mockMvc.perform(post(CREATE_UPDATE_SIGNER_ENDPOINT)
```

Comment:
In general, there is a danger not escaping the variable, you may use templating something like
```suggestion
        final var mvcResult = mockMvc.perform(post("/api/signers/{id}", DUMMY_EXTERNAL_SIGNER_ID)
```

Reply:
Good point, fixed. Just note, I did it for the `put`, in the `post` we don't use variable in path.